docs/getting_started/Demo.md

```diff
@@ -0,0 +1,347 @@
+# Pelorus Demo
+
+In this demo, you will
+
+* Get a taste of how Pelorus captures a change going through the application's delivery cycle.
+* Understand that Pelorus should be used as a conversation tool to read the trends in metrics and react by making informed investments in the software delivery process.
+* Use GitHub as the code and failure provider...TODO write better
```

Comment:
"Use GitHub as the commit exporter ans as well as the failure exporter" would be a good text?